janitor/functions/complete.py

```diff
@@ -191,9 +309,7 @@ def _complete_column(column, df):
     A Pandas Series/DataFrame with no duplicates,
     or a list of unique Pandas Series is returned.
     """
-    raise TypeError(
-        """This type is not supported in the `complete` function."""
-    )
+    raise TypeError("This type is not supported in the `complete` function.")
```

Comment:
@samukweku can we get a test that covers this line as well?

Reply:
this check is already covered in the _complete_checks section; it is safe to remove the TypeError and use a default option here.